crates/next-core/src/next_font_google/options.rs

```diff
@@ -24,32 +25,37 @@ pub struct NextFontGoogleOptions {
     pub subsets: Option<Vec<String>>,
 }
 
+impl NextFontGoogleOptionsVc {
+    pub fn new(options: NextFontGoogleOptions) -> Self {
```

Comment:
Is this supposed to be a turbo function? Right now, the cell is not cached based on `options`, so every call gets a brand new cell.